exchanges/zb/zb_wrapper.go

```diff
@@ -396,15 +397,14 @@ func (z *ZB) CancelAllOrders(_ *order.Cancel) (order.CancelAllResponse, error) {
 		Status: make(map[string]string),
 	}
 	var allOpenOrders []Order
-	enabledPairs := z.GetEnabledPairs(asset.Spot)
-	for x := range enabledPairs {
-		// Limiting to 10 pages
-		for pageNumber := int64(0); pageNumber < 11; pageNumber++ {
-			fCurr := z.FormatExchangeCurrency(enabledPairs[x], asset.Spot).String()
-			openOrders, err := z.GetUnfinishedOrdersIgnoreTradeType(fCurr,
-				pageNumber,
-				10)
+	for _, currency := range z.GetEnabledPairs(asset.Spot) {
```

Comment:
currency is a package name.
Isn't an index based loop better here? I feel like there was a nit to change it to this though.. 🤷‍♂

Reply:
Yeah this was changed after a botched rebase. Nice catch.